File: add-script-tag.js
const fs = require('fs');

const content = fs.readFileSync('web/index.html', 'utf8');

// Add the script tag after app-enhanced.js
const updated = content.replace(
    '        <script src="app-enhanced.js"></script>\n        <script>',
    '        <script src="app-enhanced.js"></script>\n        <script src="clickable-stats.js"></script>\n        <script>'
);

fs.writeFileSync('web/index.html', updated, 'utf8');

console.log("Script tag added successfully!");

File: add-script-tag.py
#!/usr/bin/env python3
with open('web/index.html', 'r', encoding='utf-8') as f:
    content = f.read()

# Add the script tag after app-enhanced.js
content = content.replace(
    '        <script src="app-enhanced.js"></script>\n        <script>',
    '        <script src="app-enhanced.js"></script>\n        <script src="clickable-stats.js"></script>\n        <script>'
)

with open('web/index.html', 'w', encoding='utf-8') as f:
    f.write(content)

print("Script tag added successfully!")
